.github/workflows/lint.yaml

```
@@ -0,0 +1,30 @@
+name: Lint Charts
+
+on:
+  pull_request:
+    paths:
+      - charts/**
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)